components/input.css

```diff
@@ -0,0 +1,40 @@
+.input{
+    display: flex;
+}
+
+.input-text{
+    flex-direction: column;
+}
+
+.input-checkbox-radio{
+    align-items: center;
+}
+
+.input label{
+    font-weight: 500;
+    margin: 0.3rem;
+}
+
+input{
+    border: 1px solid var(--lynx-gray-color);
```

Comment:
better naming can be done for color variables. good format would be --COLOR-PRIMARY like this. I am not able to understand what this lynx is

Reply:
Actually I am using different shades of gray & black, so I will rename them during refactoring.